src/js/themes/minimal.js

```diff
@@ -0,0 +1,230 @@
+        />
+      </svg>
+      <svg aria-hidden="true" viewBox="0 0 24 24" slot="exit">
+        <path
+          d="M20.25 14.5h-5a.76.76 0 0 0-.75.75v5a.75.75 0 1 0 1.5 0V16h4.25a.75.75 0 1 0 0-1.5Zm-5-5h5a.75.75 0 1 0 0-1.5H16V3.75a.75.75 0 1 0-1.5 0v5a.76.76 0 0 0 .75.75Zm-6.5 5h-5a.75.75 0 1 0 0 1.5H8v4.25a.75.75 0 1 0 1.5 0v-5a.76.76 0 0 0-.75-.75Zm0-11.5a.76.76 0 0 0-.75.75V8H3.75a.75.75 0 0 0 0 1.5h5a.76.76 0 0 0 .75-.75v-5A.76.76 0 0 0 8.75 3Z"
+        />
+      </svg>
+    </media-fullscreen-button>
+  </template>
+
+  <template partial="LiveButton">
+    <media-live-button
+      part="live seek-live button"
+      disabled="{{disabled}}"
+      aria-disabled="{{disabled}}"
+    >
+      <span slot="text" style="font-weight: normal;">Live</span>
+      <svg
+        slot="indicator"
+        width="8"
+        height="8"
+        viewBox="0 0 8 8"
+        xmlns="http://www.w3.org/2000/svg"
+        style="width: 8px; height: 8px; margin-right: 2px;"
+      >
+        <rect width="8" height="8" rx="2" />
+      </svg>
+    </media-live-button>
+  </template>
+
+  <template partial="TimeRange">
+    <media-time-range
+      part="time range"
+      disabled="{{disabled}}"
+      aria-disabled="{{disabled}}"
+    ></media-time-range>
+  </template>
+
+  <template partial="VolumeRange">
+    <media-volume-range
+      part="volume range"
+      disabled="{{disabled}}"
+      aria-disabled="{{disabled}}"
+    ></media-volume-range>
+  </template>
+
+  <media-controller>
+    <slot name="media" slot="media"></slot>
+
+    <template if="title">
+      <div slot="top-chrome">
+        <media-text-display>{{ title }}</media-text-display>
+      </div>
+    </template>
+
+    <template if="streamType == 'on-demand'">
+      <template if="breakpointSm == null">
+        <media-control-bar>
+          {{>PlayButton}}{{>TimeRange}}{{>MuteButton}}{{>CaptionsButton}}{{>FullscreenButton}}
+        </media-control-bar>
+      </template>
+
+      <template if="breakpointSm">
+        <media-control-bar>
+          {{>PlayButton}}{{>TimeRange}}
+          <template if="breakpointMd">
+            <media-time-display></media-time-display>
+          </template>
+          {{>MuteButton}}{{>VolumeRange}}{{>CaptionsButton}}{{>FullscreenButton}}
+        </media-control-bar>
+      </template>
+    </template>
+
+    <template if="streamType == 'live'">
+      <media-control-bar>
+        <div class="live-controls-left">
+          {{>LiveButton}}
+          <template if="targetLiveWindow == null">
+            <template if="breakpointSm">
+              <media-time-display></media-time-display>
+            </template>
+          </template>
+        </div>
+        <template if="targetLiveWindow > 0">
+          <template if="breakpointSm">{{>TimeRange}}</template>
+        </template>
+        <div class="live-controls-right">
+          {{>MuteButton}}{{>VolumeRange}}{{>CaptionsButton}}{{>FullscreenButton}}
+        </div>
```

Comment:
cool to see how live and DVR is mixed here in one main live view.

suggestion (non-blocking): we've been using a "spacer" div to push out left and right parts of the control bar.
it removes one level of nesting. 

see for example 
https://codesandbox.io/s/media-chrome-multi-layout-theme-gwlon8?file=/index.html